packages/kbn-ui-shared-deps-npm/webpack.config.js

```diff
@@ -103,6 +110,7 @@ module.exports = (_, argv) => {
         'rxjs/operators',
         'styled-components',
         'tslib',
+        ...euiIconEntries,
```

Comment:
I assume you tried optimization.chunks settings that didn't help?

Reply:
Yeah e.g:
```
      splitChunks: {
        minSize: 20000,
        maxAsyncRequests: 5,
      }
```